### PR TITLE
chore(genesis): update local genesis alloc and exec hash

### PR DESCRIPTION
**Pull request by 0xHansLee**
related PR: https://github.com/piplabs/story-geth/pull/48

issue: none

### CODE REVIEW

**Review by edisonz0718 — APPROVED** [review]

**Review by Ramarti — APPROVED** [review]
LGTM

File: lib/netconf/local/genesis.json [hunk 1]
```diff
@@ -88,7 +88,7 @@
     },
     "evmengine": {
       "params": {
-        "execution_block_hash": "iRNkS71nsvEdk1YbDntwiTV61iSSxe9RNKmOgVkZSPQ="
+        "execution_block_hash": "FhUhde3Ykx2/EJEuo+Cci7L/6M5g5P9dPEqXMnbmFOk="
```
Comment (limengformal) [inline lib/netconf/local/genesis.json:91]: Did we change the hash for odyssey configuration?

File: contracts/script/GenerateAlloc.s.sol [hunk 2]
```diff
@@ -377,6 +377,7 @@ contract GenerateAlloc is Script {
             vm.deal(0x00FCeC044cD73e8eC6Ad771556859b00C9011111, 100000000 ether);
             vm.deal(0xb5350B7CaE94C2bF6B2b56Ef6A06cC1153900000, 100000000 ether);
             vm.deal(0x13919a0d8603c35DAC923f92D7E4e1D55e993898, 100000000 ether);
+            vm.deal(0x64a2fdc6f7CD8AA42e0bb59bf80bC47bFFbe4a73, 100000000 ether);
```
Comment (limengformal) [inline contracts/script/GenerateAlloc.s.sol:380]: why do we add this account?

Comment (0xHansLee) [reply contracts/script/GenerateAlloc.s.sol:380]: This is my EOA for local test

File: lib/netconf/local/genesis.json (hunk 1 above)
Comment (0xHansLee) [reply lib/netconf/local/genesis.json:91]: This is for local. I updated for odyssey in [this PR](https://github.com/piplabs/story/pull/271)

**Review by limengformal — APPROVED** [review]

**Pull request merged**